OasisDEX/oasis-borrow
# Basic buy doesnt trigger bs or sl, sl doesnt trigger basic buy

# [Basic buy doesnt trigger bs or sl, sl doesnt trigger basic buy](https://app.shortcut.com/oazo-apps/story/4959/validation-basic-buy-target-ratio-does-not-trigger-basic-sell?cf_workflow=500000053&ct_workflow=all&vc_group_by=day&groupBy=workflowState)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- Warning when trying to set SL near auto-buy
![image](https://user-images.githubusercontent.com/6871923/177432686-af2e1a94-9147-4b69-bebf-7f80f331fa28.png)

- Warning when Auto-Buy may trigger Auto-Sell
- Warning when Auto-Buy may trigger Stop-Loss
  
## How to test 🧪
  Stop-loss triggering auto buy
- set auto buy
- try to set stop-loss
- expect warning when trying to set it at max value of a slider
